setup.py

```diff
@@ -43,7 +43,10 @@ def py2_only(*dependencies):
         "Programming Language :: Python :: 3.6",
         "Topic :: Software Development",
     ],
-    install_requires=["graphviz", "matplotlib"],
+    install_requires=[],
+    extras_require=[
+        "visual": ["graphviz", "matplotlib"],
```

Comment:
Should we use `"drawings"` instead of `"visual"`? This is how we refer to Gantt charts and Dependency graphs in [the doc](http://jacquev6.github.io/ActionTree/user_guide/drawings.html). At least I think we need to homogenize those.

Reply:
Agreed.